tests/snapshots/run__file@empty.hvm.snap

```diff
@@ -4,6 +4,6 @@ expression: rust_output
 input_file: tests/programs/empty.hvm
 ---
 exit status: 101
-thread 'main' panicked at src/ast.rs:508:41:
+thread 'main' panicked at src/ast.rs:527:41:
```

Comment:
off-topic but this diff is pretty annoying; we should figure out a better way to test this